bin/export.py

```diff
@@ -478,7 +478,7 @@ def update_contest_id(cid: str) -> None:
 
 
 def export_contest(cid: Optional[str]) -> str:
-    data = contest_yaml()
+    data = dict(contest_yaml())
```

Comment:
`dict(None)` crashes, I think we should cast the optional immutable `Mapping` to `dict` _after_ we check `if not data` in the lines below?

Reply:
i would have hoped that mypy catches this...

Reply:
wait, no it would have catched this :) `contest_yaml()` does never return `None`